tests/e2e/common-operator-integ-suite.sh

```diff
@@ -191,18 +214,50 @@ if [ "${SKIP_BUILD}" == "false" ]; then
     get_internal_registry
   fi
 
-  # BUILD AND PUSH IMAGE
+  # BUILD AND PUSH OPERATOR IMAGE
```

Comment:
We can probably rename this function itself as `build_and_push_operator_image` and remove the comment.

Reply:
Changed